pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -350,14 +350,14 @@ stdenv.mkDerivation (finalAttrs: {
       --replace VK_EXT_VIDEO_DECODE VK_KHR_VIDEO_DECODE
   '';
 
-  patches = map (patch: fetchpatch patch) (extraPatches
+  patches = (map (patch: fetchpatch patch) (extraPatches
     ++ (lib.optional (lib.versionAtLeast version "6" && lib.versionOlder version "6.1")
       { # this can be removed post 6.1
         name = "fix_aacps_tablegen";
         url = "https://git.ffmpeg.org/gitweb/ffmpeg.git/patch/814178f92647be2411516bbb82f48532373d2554";
         hash = "sha256-FQV9/PiarPXCm45ldtCsxGHjlrriL8DKpn1LaKJ8owI=";
       }
-    ));
+    ))) ++ lib.optionals (lib.versionAtLeast version "5" && lib.versionOlder version "6.1") [ ./remove_force_icc_lut.patch ];
```

Nit:

```suggestion
    ))) ++ lib.optionals (lib.versionAtLeast version "5" && lib.versionOlder version "6.1") [
      ./remove_force_icc_lut.patch
    ];
```

(Makes patches for future patch additions cleaner.)